clang/lib/Driver/ToolChains/AIX.cpp

```diff
@@ -479,14 +479,6 @@ static void addTocDataOptions(const llvm::opt::ArgList &Args,
       return false;
   }();
 
-  // Currently only supported for small code model.
-  if (TOCDataGloballyinEffect &&
-      (Args.getLastArgValue(options::OPT_mcmodel_EQ).equals("large") ||
-       Args.getLastArgValue(options::OPT_mcmodel_EQ).equals("medium"))) {
-    D.Diag(clang::diag::warn_drv_unsupported_tocdata);
```

Comment:
the patch also support "medium" but in the description of the patch , it is "64-bit large code-model" , and do you want to add  medium test case?

Reply:
AIX does not have medium model.